types/jira-client/index.d.ts

```diff
@@ -0,0 +1,368 @@
+    ): Promise<JiraApi.JsonResponse>;
+
+    getAllVersions(
+        boardId: string,
+        startAt?: number,
+        maxResults?: number,
+        released?: "true" | "false"
+    ): Promise<JiraApi.JsonResponse>;
+
+    private makeRequestHeader(uri: string, options?: JiraApi.UriOptions);
+
+    private makeUri(options: JiraApi.UriOptions): string;
+
+    private makeWebhookUri(options: JiraApi.UriOptions): string;
+
+    private makeSprintQueryUri(options: JiraApi.UriOptions): string;
+
+    private makeDevStatusUri(options: JiraApi.UriOptions): string;
+
+    private makeAgileUri(options: JiraApi.UriOptions): string;
+
+    private doRequest(requestOptions: CoreOptions): Promise<RequestResponse>;
+}
+
+declare namespace JiraApi {
+    interface JiraApiOptions {
+        protocol?: string;
+        host: string;
+        port?: string;
+        username?: string;
+        password?: string;
+        apiVersion?: string;
+        base?: string;
+        intermediatePath?: string;
+        strictSSL?: boolean;
+        request?: any;
+        timeout?: number;
+        webhookVersion?: string;
+        greenhopperVersion?: string;
+        bearer?: string;
+        oauth?: OAuth;
+    }
+
+    interface OAuth {
+        consumer_key: string;
+        consumer_secret: string;
+        access_token: string;
+        access_token_secret: string;
+        signature_method?: string;
+    }
+
+    interface LinkObject {
+        [name: string]: any;
+    }
+
+    interface Query {
+        [name: string]: any;
+    }
+
+    interface JsonResponse {
+        [name: string]: any;
+    }
+
+    interface UserObject {
+        [name: string]: any;
+    }
+
+    interface IssueObject {
+        [name: string]: any;
+    }
+
+    interface ComponentObject {
+        [name: string]: any;
+    }
+
+    interface FieldObject {
+        [name: string]: any;
+    }
+
+    interface FieldOptionObject {
+        [name: string]: any;
+    }
+
+    interface TransitionObject {
+        [name: string]: any;
+    }
+
+    interface WorklogObject {
+        [name: string]: any;
+    }
+
+    interface EstimateObject {
+        [name: string]: any;
+    }
+
+    interface WebhookObject {
+        [name: string]: any;
+    }
+
+    interface NotificationObject {
+        [name: string]: any;
+    }
+
+    interface BoardObject {
+        [name: string]: any;
+    }
+
+    interface SearchUserOptions {
+        username: string;
+        startAt?: number;
+        maxResults?: number;
+        includeActive?: boolean;
+        includeInactive?: boolean;
+    }
+
+    interface SearchQuery {
+        startAt?: number;
+        maxResults?: number;
+        fields?: string[];
+    }
+
+    interface SearchQuery {
+        startAt?: number;
+        maxResults?: number;
+        fields?: string[];
+    }
+
+    interface UriOptions {
+        pathname: string;
+        query?: Query;
+        intermediatePath?: string;
+    }
+}
+
+export = JiraApi;
```

Comment:
It's `export default` rather than `export =`: https://github.com/jira-node/node-jira-client/blob/master/src/jira.js#L10